### DISCUSSION TITLE
Default context view for elements

### DISCUSSION THREAD

**Discussion by axelthas** [likec4/likec4#312] [Ideas]
Hello. I have just started using likeC4 and I just have to thank you and congratulate you for this great project. I hope that it will gain attraction from software architects and developers from all around. 

This is my problem/use case:
When I am navigating through my architecture, starting from the context diagram of my system and navigating deeper into my architecture, I may stumble upon elements that have no views defined for them. As a user of the interactive UI (React) for the model, I would expect that every element is clickable and that I could see that same element in context with other elements. What currently happens in VSCode is that the code editor opens at the element that was pressed. I would maybe expect that to happen when I click the element in the default view.

So, my suggestion is to have the means to define a default view that would be used whenever there is no specific view defined for an element. I would also like to suggest having the means to navigate back from where you came from. For instance, if you end up in a default view for an element, there is no way to navigate back to the parent element unless the parent element is also displayed in the view as a container around the element where the central focus is.

Looking forward to hearing what you think.

Thanks.

**Comment by davydkov** [comment]
Hey @axelthas 

I appreciate your interest in the project.

Just to confirm I understand you right.  
You are talking about VScode diagram panel, aren't you?

**Reply by axelthas** [reply]
Yes I am. Maybe this does not relate to the generated code. I am not sure.
